### PR TITLE
Fix #1172: switch workspace on thumbnail clicked

### LINKED ISSUE #1172: Option "Click empty space to close overview" captures clicks directed at workspace switcher
Checked that the bug exists when only having d2p active.

**Describe the bug**
Turning on "Click empty space to close overview" makes it impossible to use the workspace switcher on the overview, since clicking one of the workspaces just closes the overview.

**Linux distribution and version**
Pop!_OS 20.04

**GNOME Shell version**
GNOME Shell 3.36.4

**Dash-to-Panel version**
39

**Where was Dash-to-Panel installed from?**
The GNOME extensions website

### PR DESCRIPTION
**Pull request by l3nn4rt**

### CODE REVIEW

File: overview.js [hunk 1]
```diff
@@ -446,18 +446,8 @@ var dtpOverview = Utils.defineClass({
...
-                        visibleView = index;
-                        return false;
-                    }
-                    else
-                        return true;
-                });
-
                 if(Me.settings.get_boolean('animate-show-apps')) {
-                    let view = views[visibleView].view;
+                    let view = views.find(v => v.view.actor.visible).view;
```
Comment (charlesg99) [inline overview.js:450]: Thanks l3nn4rt! Although we can't use `Array.prototype.find` as it isn't available in older gnome-shell versions :( 

For this , please use https://github.com/home-sweet-gnome/dash-to-panel/blob/8d20fb82c640b7fc4bc3edd2df015ec851c0cc39/utils.js#L324.

Comment (l3nn4rt) [reply overview.js:450]: Oh yes, I'll fix it right now! Thank you for reviewing :+1:

**Comment by charlesg99** [comment]
Thanks again!

@jderose9 With this fix in, I think we should release v40 as 3.38 has started hitting Arch repos.

**Comment by jderose9** [comment]
@charlesg99 Awesome, thanks to you both!

**Pull request merged**